plutus-core/plutus-core/test/Value/Spec.hs

```diff
@@ -188,6 +188,31 @@ prop_containsEnforcesPositivity v
       (BuiltinFailure{}, BuiltinFailure{}) -> property True
       _                                    -> property False
 
+prop_negateIsInverse :: Value -> Property
```

Comment:
You can add more tests like `prop_negateValueInvolutive` and `prop_scaleBy1IsIdentity`